<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnTransaksi">
         <property name="text">
          <string>Transaksi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnHistori">
         <property name="text">
          <string>Histori</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBudgeting">
         <property name="text">
          <string>Budgeting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPengaturan">
         <property name="text">
          <string>Pengaturan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRekomendasi">
         <property name="text">
          <string>Rekomendasi</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="Page1Histori">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Histori</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tabelHistori">
          <column>
           <property name="text">
            <string>Tanggal</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Jenis</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Kategori</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Jumlah</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Catatan</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Page4Rekom">
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Rekomendasi</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Page2Budgeting">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QPushButton" name="TambahKategori">
          <property name="text">
           <string>Tambah</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Masukkan Kategori</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QTableWidget" name="tabelBudget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::DropAction::TargetMoveAction</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Kategori</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Persentase(%)</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="SimpanBudget">
          <property name="text">
           <string>Simpan</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Budgeting</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="inputKategoriBudget"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>List Budget</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Page3Pengaturan">
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pengaturan</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Page0Transaksi">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TRANSAKSI</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Pilih Jenis Transaksi : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboJenis">
          <item>
           <property name="text">
            <string>Pemasukan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pengeluaran</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Masukkan Nominal Uang </string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="inputJumlah"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Masukkan Tanggal</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QDateEdit" name="inputTanggal"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Pilih Kategori</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QComboBox" name="comboKategori">
          <item>
           <property name="text">
            <string>Makan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Infaq/shadaqah</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Transport</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bayar Kos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UKT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Air</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Other</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Catatan</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QTextEdit" name="inputCatatan"/>
        </item>
        <item row="10" column="1">
         <widget class="QPushButton" name="SimpanTransaksi">
          <property name="text">
           <string>Simpan</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
